components/molecule/field/src/index.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import AtomValidationText, {
+  AtomValidationTextTypes
+} from '@s-ui/react-atom-validation-text'
+import AtomLabel, {AtomLabelTypes} from '@s-ui/react-atom-label'
+import AtomHelpText from '@s-ui/react-atom-help-text'
+
+const BASE_CLASS = 'sui-MoleculeField'
+
+class MoleculeField extends Component {
+  getClassNames(inline) {
+    return cx(BASE_CLASS, inline && `${BASE_CLASS}--inline`)
+  }
+
+  getTypeValidation(element) {
+    if (this.props.successText) {
+      if (element === 'label') return AtomLabelTypes.SUCCESS
+      if (element === 'validationText') return AtomValidationTextTypes.SUCCESS
+    }
+    if (this.props.errorText) {
+      if (element === 'label') return AtomLabelTypes.ERROR
+      if (element === 'validationText') return AtomValidationTextTypes.ERROR
+    }
+  }
+
+  get statusValidationText() {
+    if (this.props.successText) return this.props.successText
+    if (this.props.errorText) return this.props.errorText
+  }
+
+  render() {
+    const {
+      label,
+      helpText,
+      name,
+      inline,
+      successText,
+      errorText,
+      children // eslint-disable-line react/prop-types
+    } = this.props
+    return (
+      <div className={this.getClassNames(inline)}>
+        <AtomLabel
+          type={this.getTypeValidation('label')}
+          name={name}
+          text={label}
+        />
+        <div>
+          {children}
+          {(successText || errorText) && (
+            <AtomValidationText
+              type={this.getTypeValidation('validationText')}
+              text={this.statusValidationText}
+            />
+          )}
+          <AtomHelpText text={helpText} />
+        </div>
+      </div>
+    )
+  }
+}
+
+MoleculeField.displayName = 'MoleculeField'
+
+MoleculeField.propTypes = {
+  /** Text to be displayed as label of the textarea */
+  label: PropTypes.string.isRequired,
+
+  /** used as for attribute. Must be the same as the input element id */
+  name: PropTypes.string.isRequired,
+
+  /** Success message to display when success state  */
+  successText: PropTypes.string,
+
+  /** Error message to display when error state  */
```

Comment:
it's kind of strange as I do think about an error/success as a state, as well as you do in your comments, but it's resolved via props. Should there really be no state at all?

Reply:
Shouldn't the form including all these `Field`'s be in charge of managing when there is a  sucess/error state for each `Field`?